<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>statisticwindow</class>
 <widget class="QDialog" name="statisticwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QLabel" name="airportNameLabel">
     <property name="font">
      <font>
       <family>Arial Rounded MT</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="font">
      <font>
       <family>Arial Rounded MT</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="annualLoad">
      <property name="font">
       <font>
        <family>Arial Rounded MT</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <attribute name="title">
       <string>Загруженность за год </string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QVBoxLayout" name="annualLoadLayout"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="monthlyLoad">
      <attribute name="title">
       <string>Загруженность за месяц</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QComboBox" name="months"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="monthlyLoadLayout"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="closeWindow">
     <property name="font">
      <font>
       <family>Arial Rounded MT</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ЗАКРЫТЬ</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
